### ISSUE TITLE
Buggy code

### ISSUE THREAD

**Issue by trc492** [DrDab/TrcAttendance-Android#10]
Don't know what this code is doing but my common sense told me this code doesn't look right.
OnActivityResult in MainActivity.java:

```
    // add new attendants into the list.
    for(int i = 0; i < DataStore.attendanceLog.attendantsList.size(); i++)
    {
        DataStore.allAttendants = new ArrayList<Attendant>();
        Attendant lol = DataStore.attendanceLog.attendantsList.get(i);
        DataStore.allAttendants.add(lol);
        if(DataStore.checkInList.indexOf(lol) == -1)
        {
            DataStore.checkInList.add(lol);
        }
    }
```

Inside the for-loop, you are creating a new ArrayList overwriting DataStore.allAttendants in every loop. Then you added only ONE element to the array, loop again, created another new ArrayList, added the next ONE element to the new array and repeat …

**Comment by DrDab** [comment]
Yep, I agree, this isn't very efficient. I will setup something more organized as soon as possible.

**Comment by DrDab** [comment]
Yep, typo fixed. This should be resolved as of commit c336a60. Don't know what that code was doing in there in the first place, likely an unused code stub for something else.
